src/Database/PostgreSQL/PQTypes/DerivingVia.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE AllowAmbiguousTypes, TypeApplications #-}
+module Database.PostgreSQL.PQTypes.DerivingVia (
```

Comment:
Would `Deriving` be better than `DerivingVia` here?

Reply:
I agree. Changed!